spec/features/create_dataset_spec.rb

```diff
@@ -22,7 +22,7 @@
     end
 
     it "publishes a dataset" do
-      visit "/datasets/new"
```

Comment:
removing hard-coded URLs means those could be modified in the future without having to update the tests each time.